whois-db/src/main/resources/applicationContext-whois-base.xml

```diff
@@ -15,8 +15,7 @@
 
     <import resource="applicationContext-nrtm.xml"/>
 
-    <import resource="applicationContext-nrtm4.xml"/>
```

Comment:
Is it possible to isolate the NRTMv4 changes into a separate PR ?

Reply:
These are all inter connected and actually easy to move.  The ones which requires cnsiderable work and some brains I have decided to put them in different PR

Reply:
separated https://github.com/RIPE-NCC/whois/pull/1504